src/css/custom.css

```diff
@@ -195,6 +195,11 @@ svg.implemented:hover [stroke="#333"] {
   margin-bottom: 1em;
 }
 
+.badge--beta {
+  background-color: #26d07c;
```

Comment:
FYI: Feel free to choose a different color here.

Reply:
This one works well I guess